cogment_verse/model_registry_v2.py

```diff
@@ -0,0 +1,556 @@
+# Copyright 2022 AI Redefined Inc. <dev+cogment@ai-r.com>
```

Comment:
Why copying the cogment file? Why not use cogment?

Reply:
And why do you need the stub directly?

Reply:
Alright, I'm totally for that.